### PR TITLE
fix to use latest markdown and add tests

### PR DESCRIPTION
**Pull request by IvanovIvan1900**

### CODE REVIEW

**Comment by uhurusurfa** [comment]
Besides th fact that Pinax is not Django 4 compatible, is there another reason why you want to restrict the app to Django 3?
Also, for markdown... are you seeing problems using it with the latest version and if so where?

**Comment by IvanovIvan1900** [comment]
this is project, i think, not compatible witch Django 4 see problem
https://github.com/django-helpdesk/django-helpdesk/issues/1020

markdown
install markdown latest version, but in version 3.0 deprecated md_global and if you open ticket to get error.
https://github.com/Python-Markdown/markdown/blob/master/docs/change_log/release-3.0.md?ysclid=l5v68kuq2p348423531

**Comment by uhurusurfa** [comment]
Thanks for the detailed feedback. There must be a missing unittest to catch the markdown problem and will need to add the unit test along with a fix for using markdown 3.
If you are up for fixing the mardown isue and  adding a unit test please feel free to raise a PR otherwise someone will pick that up soon.

**Comment by IvanovIvan1900** [comment]
add the comment witch tests

**Comment by uhurusurfa** [comment]
@IvanovIvan1900  - please remove the restriction on the markdown version and remove the md_globals from the extendMarkdown() method and this should pass the unit tests and remove the issue with Markdown versions.

**Comment by IvanovIvan1900** [comment]
Hi. only deletion "md_globals" not working. I am rewrite model.py, test and requirments. Now work witch latest markdown.

**Comment by uhurusurfa** [comment]
Please change your PR to merge into the "unstable" branch.

**Comment by IvanovIvan1900** [comment]
> Please change your PR to merge into the "unstable" branch.

Done

**Comment by gwasser** [comment]
Looks like there's some conflicts, you might need to update this branch with a pull from unstable.

**Comment by IvanovIvan1900** [comment]
> Looks like there's some conflicts, you might need to update this branch with a pull from unstable.

Hi, i merge commit and correct coflict.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Django>=3.2
+Django>=3.2,<4.0
```
Comment (uhurusurfa) [inline requirements.txt:1]: You need to remove this restriction: <4.0
It is causing the CI/CD pipeline to fail to install Django.
We will fix the rundemo feature in the near future but the project must support Django 4 now.

**Review by uhurusurfa — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (IvanovIvan1900) [reply requirements.txt]: Jos is done

**Review by gwasser — APPROVED** [review]
thank you, some test cases for the markdown code are helpful.

**Comment by kylecaston** [comment]
@gwasser @uhurusurfa I just came across this issue for one of my projects.  Although the fix was merged into what's now the `main` branch, a new version was never published to PyPi so projects are still picking up the broken version.  Could you please publish a new version?

**Comment by gwasser** [comment]
Sure I can publish a new version if @uhurusurfa agrees. I haven't paid attention to recent changes so am not sure if it's a stable position for publishing or if more work is needed.

**Pull request merged**